developers/integrations/llm-frameworks/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: LLM Frameworks
+sidebar_position: 4
+image: og/integrations/home.jpg
+---
+
+Large Language Model (LLM) frameworks contain several tools to help developers build applications with generative models.
```

Comment:
Consider
These Large Language Model (LLM) frameworks have tools to help developers build applications with generative models.